### ISSUE TITLE
Skip link not appearing on mobile screen readers

### ISSUE THREAD

**Issue by HarveyPeachey** [bbc/simorgh#7478]
**Describe the bug**
When carrying out the a11y swarm for Ukrainian and Ukrainian in Russian we found skip links for the navigation bar, social media embeds and includes aren't showing visually for mobile screen readers, but instead highlighting the area where the skip link is meant to be and reading out the skip link text.

Note: This is not exclusive to the Ukrainian service.

**To Reproduce**
Steps to reproduce the behaviour:

1. Enable Voice Over on iOS or Talkback on android
2. Go to https://www.test.bbc.com/ukrainian/news-russian-23333960 or any test asset page on any service language
3. Scroll through to a skip link

**Expected behaviour**
The skip link should appear for the user when selected.

**Screenshots**
iOS Voice Over
![Image from iOS (1)](https://user-images.githubusercontent.com/4706195/90037284-f1167f80-dcbb-11ea-9256-28a632ffd779.png)

Android Talkback
![image (2)](https://user-images.githubusercontent.com/4706195/90037299-f673ca00-dcbb-11ea-95d4-7178a956adf0.png)

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**

**Comment by greenc05** [comment]
Closing this issue. The skip link should be visually dispayed for keyboard users, visually displaying the content on focus isn't expected for mobile screen readers.
